linux_os/guide/system/auditing/auditd_configure_rules/file_permissions_var_log_audit/bash/shared.sh

```diff
@@ -6,7 +6,7 @@ else
     FILE="/var/log/audit/audit.log"
 fi
 
-{{% if product not in ["ol8", "rhel8"] %}}
+{{% if product not in ["ol8", "rhel7", "rhel8", "rhel9"] %}}
```

Comment:
```suggestion
{{% if product not in ["ol8" ] and 'rhel' not in product %}}
```
Or something similar. So that we don't have to do this again, in the future.